tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -188,28 +188,58 @@ install_initrd_sev() {
 	install_initrd "initrd-sev" "sev"
 }
 
+#Install kernel component helper
+install_cached_kernel_tarball_component() {
+	local kernel_name=${1}
+	local job_name=${2}
+
+	install_cached_tarball_component \
+		"${kernel_name}" \
+		"${jenkins_url}/job/${job_name}/${cached_artifacts_path}" \
```

Comment:
I'm still not convinced about the `job_name` change.

Reply:
I agree with you here. I have removed the job_name changes. They are not required.